src/modules/SX126x.h

```diff
@@ -152,6 +152,8 @@
 //SX126X_CMD_SET_PA_CONFIG
 #define SX126X_PA_CONFIG_HP_MAX                       0x07
 #define SX126X_PA_CONFIG_PA_LUT                       0x01
+#define SX126X_PA_CONFIG_SX1261                       0x01
```

Comment:
Move the `SX126X_PA_CONFIG_SX1261` macro to SX1261.h, that's more appropriate for it.